SchoolRush/Assets/Scripts/GameController/HUDController.cs

```diff
@@ -63,6 +67,19 @@ void Update()
                 checkpointLocationText.text = "302동";
                 break;
         }
+
+        // update status
+        Color32 offColor = new Color32(0x50, 0x50, 0x50, 0xff);
+        if (kartController.GetCanJump()){
+            jump.color = Color.white;
+        } else jump.color = offColor;
+        if (kartController.GetCanBoost()) {
+            boost.color = Color.white;
+        } else boost.color = offColor;
+        if (kartController.GetIsDrifting()) {
+            drifting.color = Color.white;
+        } else drifting.color = offColor;
```

Comment:
```suggestion
        if (kartController.GetCanJump()) jump.color = Color.white;
        else jump.color = offColor;
        
        if (kartController.GetCanBoost()) boost.color = Color.white;
        else boost.color = offColor;
        
        if (kartController.GetIsDrifting()) drifting.color = Color.white;
        else drifting.color = offColor;
```

취향일 수 있는데 포매팅이 조금 헷갈려서 수정하겠습니다!